modules/videoModule/adQueue.js

```diff
@@ -9,13 +9,13 @@ export function AdQueueCoordinator(videoCore, pbEvents) {
     videoCore.onEvents([SETUP_COMPLETE], onSetupComplete, divId);
   }
 
-  function queueAd(adTagUrl, divId, options) {
+  function queueAd(adXml, divId, options) {
     const queue = storage[divId];
     if (queue) {
-      queue.push({adTagUrl, options});
-      triggerEvent(AUCTION_AD_LOAD_QUEUED, adTagUrl, options);
```

Comment:
this is breaking the event signature; the param is `adTagUrl` when the queued ad is an ad tag URL. Clients could be  expecting this value.  Why is this Queueing system being changed to only work with Ad XML ?